<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>500</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>1</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
    <underline>false</underline>
    <strikeout>false</strikeout>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QWidget" name="widget_4" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <pointsize>18</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
              <underline>false</underline>
              <strikeout>false</strikeout>
             </font>
            </property>
            <property name="text">
             <string>账号：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="accountInfo">
            <property name="font">
             <font>
              <pointsize>18</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
              <underline>false</underline>
              <strikeout>false</strikeout>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <pointsize>18</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
              <underline>false</underline>
              <strikeout>false</strikeout>
             </font>
            </property>
            <property name="text">
             <string>密码：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="passwdInfo">
            <property name="font">
             <font>
              <pointsize>18</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
              <underline>false</underline>
              <strikeout>false</strikeout>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QWidget" name="widget_3" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="loginBtn">
         <property name="font">
          <font>
           <pointsize>20</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
           <underline>false</underline>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="styleSheet">
          <string>background:rgb(130, 130, 130);
color:rgb(245, 255, 55)</string>
         </property>
         <property name="text">
          <string>登录</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="regisBtn">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="font">
          <font>
           <pointsize>20</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
           <underline>false</underline>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background:rgb(130, 130, 130);
color:rgb(245, 255, 55)</string>
         </property>
         <property name="text">
          <string>注册</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QWidget" name="widget_5" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="customLabel" name="headimg">
         <property name="minimumSize">
          <size>
           <width>110</width>
           <height>110</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>110</width>
           <height>110</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>8</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
           <underline>false</underline>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>422</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>customLabel</class>
   <extends>QLabel</extends>
   <header location="global">customlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
